### DISCUSSION TITLE
page 폴더구조 변경

### DISCUSSION THREAD

**Discussion by NamgyungKim** [prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung#121] [Ideas]
## page 폴더구조 변경
- appLayout이 있지만 내부에 추가적으로 css 수정이 필요할것 같습니다. 
- Profile페이지와 Main페이지는 Applayout마저 없어서 페이지 내부에 css를 수정할 수 없게되면 불편할것 같습니다.

```
📂 pages
├── 📂creat
│       ├── inedx.tsx
│       ├── style.tsx
│       └── ???
├── 📂detail
│       ├── inedx.tsx
│       ├── style.tsx
│       └── ???
.
.
.
```

- [ ] 그대로 가자 😱
- [x] 변경하자 😍

**Comment by YuHyun-P** [comment]
저도 페이지 레이아웃 스타일을 위해서 변경해야 할 것 같습니다
